frontend/server/src/DAO/Identities.php

```diff
@@ -114,19 +114,49 @@ public static function resolveAssociatedIdentity(
         if (is_null($currentIdentity->user_id)) {
             return null;
         }
-        $sql = 'SELECT
-                    i.*
-                FROM
-                    Identities i
-                INNER JOIN
-                    Emails e
-                ON
-                    e.user_id = i.user_id
-                WHERE
-                    i.user_id = ?
-                    AND (i.username = ? OR e.email = ?)
+        $sql = '(
+                    SELECT
+                        i.*
+                    FROM
+                        Identities i
+                    INNER JOIN
+                        Emails e
+                    ON
+                        e.user_id = i.user_id
+                    WHERE
+                        i.user_id = ?
+                        AND (i.username = ? OR e.email = ?)
+                )
+                UNION
+                (
+                    SELECT
+                        i.*
+                    FROM
+                        Identities i
+                    INNER JOIN
+                        Teams t
+                    ON
+                        i.identity_id = t.identity_id
+                    INNER JOIN
+                        Team_Users tu
+                    ON
+                        tu.team_id = t.team_id
+                    INNER JOIN
+                        Users u
+                    ON
+                        tu.user_id = u.user_id
+                    WHERE
+                        u.user_id = ?
```

Comment:
```suggestion
                        i.user_id = ?
```

por consistencia con arriba.

Reply:
Esto no sería posible porque las identidades que se crean para equipos no almacenan un `user_id` en la tabla de `Identities`, para eso es el `JOIN` a la tabla de `Team_Users`. Lo más que pude hacer es quitar el último `JOIN` y que la condición se haga con `tu.user_id`